machine_usage/views.py

```diff
@@ -290,14 +292,27 @@ def generate_failed_usage_form(request):
 
         return render(request, 'machine_usage/forms/failed_usage.html', {"machines_in_use":output})
     else:
+        if(not verify_key(request)):
+            return HttpResponse("Invalid or missing API Key", status=403)
+
         machine_name = request.POST["machine_name"]
         machine = Machine.objects.get(machine_name=machine_name)
 
-        usage = machine.current_job
-        usage.clear_time = timezone.now()
-        usage.failed = True
-        usage.status_message = "Failed."
-        usage.save()
+        fail_log = {
+            "machine": machine_name,
+            "user": request.user.get_username(),
+            "filament": request.POST["filament_type"],
+            "percentage": request.POST["percentage"],
+            "error_msg": request.POST["error_msg"],
+            "observed_failure": request.POST.getlist("failure_type")
+        }
+
+        r = requests.post(settings.FAILURE_FORM_URL, json = fail_log)
```

Comment:
So sorry I didn't notice this on the first pass but could you do something like the following in case your api call fails so its still returns successfully to the user
```
try:
    r = requests.post(settings.FAILURE_FORM_URL, json = fail_log)
except requests.exceptions.RequestException as e:  # This is the correct syntax
    print(e)
```